guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
+**Lightning network routing fees**
 
-Applications sometimes also allow the sender to select fractions of their total available balance by providing a "max" or "use full balance" button. These fractions can be handy when the sender needs to make transfers from their hot wallet to a more secure hardware wallet.
-</div>
+On the Lightning network, payments are passed between nodes to get from the sender to the receiver. Each of those nodes may charge a base fee and a second fee based on a percentage of the amount forwarded. Fees paid can vary, but are typically in the single-digit or double-digit Satoshi range (a small fraction of on-chain fees).
+
+**Lightning service provider fees**
 
-**Do's**
+In certain situations, the Lightning wallet may not have enough liquidity to send or receive a payment. Wallet providers may offer to alleviate these friction points, and earn additional fees. A common scenario is the automatic opening of a payment channel when a wallet receives the first deposit.
 
-- Allow senders to switch which denomination they are inputting the amount with
-- State the total balance available (subtracting the required fee)
-- Allow selecting max amount or fractions for easier fund management
-- Payment links and QR Codes can contain an amount. When they do, your application should populate the amount field automatically
-- Indicate if an amount entered is more than the available balance
+**On-chain fees**
 
-## Setting the transaction fee
+This fee is dependent on how many other transactions are currently waiting to be processed on the base layer as a whole. The [average fee](https://ycharts.com/indicators/bitcoin_average_transaction_fee) in January 2021 was $0.63, and $28.60 in April 2021.
+
+</div>
+
+## Review & approval
 
 <div class="center" markdown="1">
+
 {% include image.html
-   image = "/assets/images/guide/payments/send/select-fee-rate.png"
-   retina = "/assets/images/guide/payments/send/select-fee-rate@2x.png"
-   alt-text = "Fee selection component with urgent, normal, and not urgent options"
-   width = 400
-   height = 470
-   caption = "G = Haitian Gourde, the local currency of Haiti."
-   layout = "float-right-desktop"
+   image = "/assets/images/guide/payments/send/confirm.png"
+   retina = "/assets/images/guide/payments/send/confirm@2x.png"
+   alt-text = "Example image"
+   caption = "A compact summary to confirm the information is accurate."
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
 %}
 
-The application can automatically estimate a fee and set it for the sender. This would typically prioritize the transaction to be included in a block as soon as possible. Since the fee rate may vary if the network is busy, you can give senders more fine-grained fee controls so they can choose to optimize for faster confirmation or lower fees.
+Particularly with payments for larger amounts, it is good practice to allow users to review and confirm the information before it is sent.
```

Comment:
I think this should be for any payment amount, bitcoin payments are final once sent and should never skip some kind of review and confirm.

Reply:
Might need to rephrase this. When you just clicked a Lightning invoice for 10 sats, all you do is review and hit "Pay". That's where no additional review screen after the invoice presentation is necessary (IMHO), it would just be duplication.

Reply:
It would just go straight to a review page wouldn't it? So it still has the same final screen for any payment amount.

Reply:
I think we're saying the same thing?